WarehouseManagement.Core/Services/MarkerService.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Microsoft.EntityFrameworkCore;
+using WarehouseManagement.Core.Contracts;
+using WarehouseManagement.Core.DTOs.Marker;
+using WarehouseManagement.Infrastructure.Data.Common;
+using WarehouseManagement.Infrastructure.Data.Models;
+
+namespace WarehouseManagement.Core.Services;
+
+public class MarkerService : IMarkerService
+{
+    private readonly IRepository repository;
+
+    public MarkerService(IRepository repository)
+    {
+        this.repository = repository;
+    }
+
+    public async Task AddAsync(MarkerFormDto model, string userId)
+    {
+        if (model != null)
```

Comment:
not 100% sure if we really need this check, but if we gonna leave it out maybe throw an error if model is null?

Reply:
Yes you are right. We dont need this check, cuz the framework do it for us.